weave/trace_server/opentelemetry/python_spans.py

```diff
@@ -268,18 +281,48 @@ def as_dict(self) -> dict[str, Any]:
     def to_call(
         self, project_id: str
     ) -> tuple[tsi.StartedCallSchemaForInsert, tsi.EndedCallSchemaForInsert]:
-        summary_insert_map = self.attributes.get_weave_summary()
-        inputs = self.attributes.get_weave_inputs()
-        outputs = self.attributes.get_weave_outputs()
-        attributes = self.attributes.get_weave_attributes(
-            extra={"otel_span": self.as_dict()}
+        events = [SpanEvent(e.as_dict()) for e in self.events]
+        usage = get_weave_usage(self.attributes) or {}
+        inputs = get_weave_inputs(events, self.attributes) or {}
+        outputs = get_weave_outputs(events, self.attributes) or {}
+        attributes = get_weave_attributes(self.attributes) or {}
+        llm_usage = tsi.LLMUsageSchema(
```

Comment:
not to future devs: llm_usage is not auto rolled up here like it is in python clients which will create different UI experiences.